### ISSUE TITLE
Write unit tests

### ISSUE THREAD

**Issue by aniri** [code4romania/mon-vot-android-kotlin#91]

**Comment by pedrofsn** [comment]
@aniri write unit tests to entire application it's a huge task. Can I create some issues to specific classes to work on this tasks? 
Ex.: Write tests to Guide area (Fragment and ViewModel)

**Comment by aniri** [comment]
@pedrofsn yes, you are right, this should be split in smaller tasks. If you could write unit tests for a few areas, that would be great!
